### ISSUE TITLE
People badges on the sidebar are not working anymore

### ISSUE THREAD

**Issue by Growiel** [dwainscheeren/dwains-lovelace-dashboard#698]
Hello there,

On the left side of the dashboard, we can find people badges.

They usually say things like "Home" or "Away" or the zone name the person are in. You could also customize the popup card that you get when clicking on the badges.

All of this is not working anymore.

Now we get "home" or "not_home" only and the popcard is showing the default one instead of the one configured in devices -> person.

This is what I have in devices -> person -> edit popup card:

![image](https://user-images.githubusercontent.com/5228735/233334180-76024a85-1781-4002-8352-7818c6a29b18.png)

It's the "history" graph along with a map that shows 3 hours of location.

but when i click on the card itself, either in devices -> person or on the badge on the left side, i get the default popup card:

![image](https://user-images.githubusercontent.com/5228735/233334512-0eea19cb-0769-460e-8c2d-acaf30a55688.png)
 (hiding relevant part of the map for obvious privacy reasons)

I feel like this happened since HA 2023.4 but I can't be 100% sure.

Thanks !

**Comment by massimoha** [comment]
Yep, mine has done this too since the update a couple days ago. shows home/not_home instead of Home/Away.

**Comment by mouledla** [comment]
Same here. Also, the whole ustom pop up does not work anymore since update to 2023.4

**Comment by hackathy** [comment]
Can confirm same behaviour on Unraid Docker with the lscr.io/linuxserver/homeassistant image from Community Applications.

**Comment by NXT44** [comment]
Same issue here "not_home"

**Comment by dwainscheeren** [comment]
Thanks for reporting this.

Dwains Dashboard 3.9.0 has now been released, and the source files for DD3 are available in this repository.

DD3 is no longer under active feature development and will slowly be phased out. Active development continues in Dwains Dashboard Next:
https://github.com/dwainscheeren/dwains-dashboard-next

At the time of writing, Dwains Dashboard Next is still in alpha.

If this issue is a feature request, please check whether it fits Dwains Dashboard Next instead.

If this issue is a bug, please retest with DD3 version 3.9.0. If the bug is still reproducible there, please open a new issue with:

- Home Assistant version
- Dwains Dashboard version
- browser and device
- browser console errors
- clear reproduction steps

For more context, see #906.

For that reason, this older issue is being closed as part of the DD3 issue tracker cleanup.
